src/pages/guidelines/accessibility/overview.mdx

```diff
@@ -120,32 +164,32 @@ Color-blindness affects 8% of all men and 0.4% of women.
 
 #### What designers should think about
 
-- Design for good keyboard interaction, making sure all actions are keyboard accessible and efficient.
-- Learn how to navigate using a [keyboard](https://help.gnome.org/users/gnome-help/stable/keyboard-nav.html.en) and spend some time navigating the web, email, and digital products using only the keyboard.
+- Design for keyboard interaction
+- Learn how to navigate using a [keyboard](https://help.gnome.org/users/gnome-help/stable/keyboard-nav.html.en) and spend some time navigating the web and digital products like email using only the keyboard.
 
 #### How this applies to everyone
 
 - Many users prefer to navigate interfaces with a keyboard and no mouse for efficiency. Good keyboard navigation can help everyone be more productive.
 
 ### Users with cognitive disabilities
 
-These users offer a broad spectrum of concerns for design. Functional cognitive disabilities can include difficulty with:
+Functional cognitive disabilities can result in difficulties with:
 
 - Memory
 - Problem solving
 - Attention
 - Reading, linguistic, and verbal comprehension
-- Math comprehension
+- Mathematics
 - Visual comprehension
 
 #### How they experience an interface
 
 - May have limited working memory and need information to remain visible throughout the completion of a task
-- May experience seizures when exposed to flashing content (epilepsy)
+- May experience seizures when exposed to flashing content due to epilepsy
 
 #### What designers should think about
 
-- Designers should understand the danger of complex language, non-stopping or flashing animations and fundamental cognitive usability heuristics.
+- Designers should understand the danger of complex language, flashing animations, and fundamental cognitive usability heuristics.
 - Design in as linear a fashion as possible and focus on design heuristics that have to do with cognitive load and memory.
 
```

Comment:
From a cognitive standpoint autoplaying gifs and videos should be avoided if used at all.